app/views/listings/show.haml

```diff
@@ -105,7 +105,8 @@
             .listing-fb-like-button
               = facebook_like(current_user?(@listing.author))
             .listing-tweet-button
-              = link_to("", "https://twitter.com/share", :class => "twitter-share-button", "data" => {count: "horizontal", via: (@current_community.twitter_handle || "Sharetribe"), text: @listing.title })
+              - twitter_params =  {count: "horizontal", via: (@current_community.twitter_handle), text: @listing.title }.compact
```

Comment:
@sktoiva did you test that this behaves nicely also in case where @current_community.twitter_handle is not set? (and probably nil)

If yes, then all the code looks good to me. 👍